Spec/Auth.swift

```diff
@@ -185,6 +185,67 @@ class Auth : QuickSpec {
                         expect(client.auth.method).to(equal(ARTAuthMethod.Token))
                     }
                 }
+
+                // RSA4b
+                it("in REST, if the token creation failed or the subsequent request with the new token failed due to a token error, then the request should result in an error") {
+                    let options = AblyTests.clientOptions()
+                    options.key = AblyTests.commonAppSetup().key
+                    options.token = getTestToken()
```

Comment:
I am not following why you provide a token.  This spec states "When the client does have a means to renew the token automatically" yet you've provided an explicit token and thus no means to renew the token?